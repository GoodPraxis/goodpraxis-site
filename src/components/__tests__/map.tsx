import * as renderer from 'react-test-renderer';
import * as React from 'react';
import Map, { MapContainer } from '../map';

describe('Map', () => {
  it('renders loading container correctly', () => {
    const tree = renderer
      .create(<Map />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders map container correctly', () => {
    const tree = renderer
      .create(<MapContainer google={null} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
