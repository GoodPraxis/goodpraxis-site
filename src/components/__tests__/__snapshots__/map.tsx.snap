// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders loading container correctly 1`] = `
<div>
  Loading...
</div>
`;

exports[`Map renders map container correctly 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "position": "absolute",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "bottom": 0,
        "display": "inherit",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    Loading map...
  </div>
</div>
`;
